docs/guides/fetch-data/01-read-profile-data.md

```diff
@@ -0,0 +1,527 @@
+---
+sidebar_label: 'Read Profile Data'
+sidebar_position: 3.1
+---
+
+# Read Universal Profile Data
+
+In this guide, we will learn how to:
+
+- call a Universal Provile address
+- read data from a Universal Profile
+
+![Universal Profile example on universalprofile.cloud](./img/example-up.png)
+
+We will use:
+
+- [web3.js](https://web3js.readthedocs.io/en/v1.7.0/) for utility as well as connecting with the LUKSO network
+- [erc725.js](../../tools/erc725js/getting-started/) library to check the interface of an profile.
+- [isomorphic-fetch](https://github.com/matthew-andrews/isomorphic-fetch) to enable you to use fetch in plain Node code
+
+## Table of Contents
+
+1. Check for valid Blockchain Address
+2. Call the Universal Profile
+3. Read the Dataset
+   1. Fetch the profile's metadata
+   2. Fetch the profile's picture properties
+   3. Fetch the profile's universal receiver
+
+## Step 1 - Check for valid Blockchain Address
+
+When receiving data from a Universal Profile within an application, it is commonly used as input. You want to make sure that the address is valid in the first place. You can do a checkup by converting the value into a checksum address from the `web3.js` library. We need to install the library first. You can do this by opening up the terminal in the project's folder of your choice.
```

Comment:
Probably stick to "We", since you started like this in the beginning.

_Suggested changes:_
"When receiving data from a Universal Profile within an application, it is commonly used as input. We want to make sure that the address is valid in the first place. We can do a checkup by converting the value into a checksum address using the `web3.js` library. 

First, open a terminal in the project's folder of your choice and install the `web3.js` library."